paasta_tools/prune_completed_pods.py

```diff
@@ -0,0 +1,130 @@
+import argparse
+import logging
+import sys
+from datetime import datetime
+from typing import Sequence
+
+from dateutil.tz import tzutc
+from kubernetes.client import V1Pod
+
+from paasta_tools.kubernetes_tools import get_all_pods
+from paasta_tools.kubernetes_tools import get_pod_condition
+from paasta_tools.kubernetes_tools import is_pod_completed
+from paasta_tools.kubernetes_tools import KubeClient
+
+log = logging.getLogger(__name__)
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Terminates completed pods based on namespace and minutes since completion"
+    )
+    parser.add_argument(
+        "-n",
+        "--namespace",
+        help="Namespace of the pods to terminate from",
+        required=True,
+    )
+    parser.add_argument(
+        "-m",
+        "--minutes",
+        default=0,
+        help="Minutes since the pods' completion. Terminates pods based on time since completion.",
+        required=True,
+    )
```

Comment:
```suggestion
    parser.add_argument(
        "-m",
        "--minutes",
        default=0,
        help="Minutes since the pods' completion. Terminates pods based on time since completion.",
        required=True,
        type=int,
    )
```

should allow us to get rid of the cast that we do on line 90

Reply:
Done